grpc/streambuffer/streambuffer.go

```diff
@@ -126,7 +139,7 @@ func (s *Stream) Run() (err error) {
 			if s.recvFunc != nil {
 				r = s.recvFunc()
 			} else {
-				r = new(empty.Empty)
+				r = new(empty.Empty) // Generic proto message if not interested in received values
```

Comment:
what I meant was documenting that `Run()` unmarshals the received message into `empty.Empty`